sunpy/coordinates/tests/test_frames.py

```diff
@@ -427,3 +427,18 @@ def test_skycoord_hpc(args, kwargs):
         hgs = sc.transform_to("heliographic_stonyhurst")
 
     assert isinstance(hgs.frame, HeliographicStonyhurst)
+
+
+def test_ang_rad():
```

Comment:
You can use full words, the test names don't have to be short.

Reply:
ohh, ok!